blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,225 @@
+[support for FreeBSD](https://github.com/fwupd/fwupd/pull/3330). These changes
+made it possible to open fwupd to another group of operating systems.
+These changes were worked on by Michal Kopeć, Sergii Dmytruk, and Norbert Kamiński.
+
+Another sizable change that our engineers were responsible for was extending
+[flashrom support](https://github.com/fwupd/fwupd/commit/2bac03eee1a68becc96712d3a86656cc2e8fa19d)
+and adding support for Tuxedo laptops.
+
+Our team can help you seamlessly integrate into the fwupd ecosystem across
+a variety of platforms and operating systems. Say goodbye to clunky update
+processes and hello to a streamlined, hassle-free experience with fwupd!
+
+---
+
+![yocto logo](/img/YoctoProject_Logo_RGB.jpg)
+
+If you're looking for a tailored Linux-based operating system that perfectly
+meets your unique requirements and security needs, Yocto is the open-source
+project for you. As a comprehensive suite of tools and templates, Yocto
+provides the flexibility and customization you need to create a bespoke
+solution for your device.
+
+Our engineers have added some important fixes to some of Yocto's most popular
+layers. Tomasz Żyjewski added support for the Dunfell version of
+[meta-openwrt](https://github.com/kraj/meta-openwrt/pull/113)
+as well as added support for
+[python3-binwalk](https://github.com/openembedded/meta-openembedded/commit/7ea0e04aaee19f61e18bf998bb07f02e52d8146e)
+and [python3-uefi-firmware](https://github.com/openembedded/meta-openembedded/commit/3e70428db7b48573883aa50de636d93757dd263e)
+in the `meta-openembedded` layer.
+
+Cezary Sobczak added support for the
+[Nezha Allwinner D1](https://github.com/riscv/meta-riscv/pull/327) in
+the `meta-riscv` layer. And Maciej Pijanowski added minor fixes for
+the [meta-sunxi](https://github.com/linux-sunxi/meta-sunxi/pull/256) layer.
+
+By partnering with our team, we can help you leverage the power of Yocto and
+build a custom Linux distribution that fully aligns with your vision. From
+feature-rich IoT devices to mission-critical servers, we've got you covered.
+Let us create a personalized solution that meets your exact specifications
+and takes your device's capabilities to the next level.
+
+![Trenchboot logo](/img/trenchboot_logo.svg)
+
+[TrenchBoot](https://blog.3mdeb.com/tags/trenchboot/) is a framework that
+allows individuals and projects to build security engines to perform launch
+integrity actions for their systems. The framework builds upon Boot Integrity
+Technologies (BITs) that establish one or more Roots of Trust (RoT) from which
+a degree of confidence that integrity actions were not subverted is derived.
+
+The biggest changes took place in the landing-zone component. Worth mentioning
+is the addition of [support for the Xen hypervisor](https://github.com/TrenchBoot/landing-zone/pull/60)
+and [multiboot2 support for the GRUB2 bootloader](https://github.com/TrenchBoot/landing-zone/pull/64).
+The author of these changes is Krystian Hebel.
```

Comment:
Link profile.

Reply:
Link added https://github.com/3mdeb/news-and-ideas/pull/519/commits/68a72cdebcee326365faf61a3b7861f718765122#diff-5dff579917e1df023c3d68f093f637f7fb497ee2f22650cc786ac30a195a7eceR311